docs/malformed_files.md

```diff
@@ -0,0 +1,79 @@
+# Dealing with malformed files
+
+
+## Field compatibility
+
+When loading multiple files, the `INFO` and `FORMAT` fields
+from all VCF files are merged together to generate a "representative header",
+which is then used to generate a single BigQuery schema. If the same key is
+defined in multiple files, then its definition must be compatible across files.
+The compatibility rules are as follows:
+
+* Fields are compatible if they have the same `Number` and `Type` fields.
+  Annotation fields (i.e. those specified by `--annotation_fields`) must also
+  have the same `Description`.
+
+* Fields with different `Type` values are compatible in the following cases:
+
+  * `Integer` and `Float` fields are compatible and are converted to `Float`.
+  * You must run the pipeline with --allow-incompatible-records to
+    automatically resolve conflicts between incompatible fields (e.g. String
+    and Integer). This is to ensure incompatible types are not silently ignored.
+    See [below](#specifying---allow-incompatible-records) for more details.
+
+* Fields with different `Number` values are compatible in the following cases:
+
+  * "Repeated" numbers are compatible with each other. They include:
+    * `Number=.` (unknown number)
+    * Any `Number` greater than 1.
+    * `Number=G` (one value per genotype) and `Number=R` (one value for each
+      alternate and reference).
+    * `Number=A` (one value for each alternate) only if running with
+      `--split_alternate_allele_info_fields False`.
+
+  * You must run the pipeline with --allow-incompatible-records to
```

Comment:
I find there are multiple places for the flags that are using `-` instead of the underline. This is one example of that.

Reply:
Also please add `` for the flag here.